qa/rpc-tests/walletbackup.py

```diff
@@ -76,9 +76,9 @@ def setup_network(self, split=False):
 
         # nodes 1, 2,3 are spenders, let's give them a keypool=100
         extra_args = [
-           ['-debug=zrpc', "-keypool=100", ed0],
-           ['-debug=zrpc', "-keypool=100", ed1],
-           ['-debug=zrpc', "-keypool=100", ed2],
+           ['-debug=1', "-keypool=100", ed0],
+           ['-debug=1', "-keypool=100", ed1],
+           ['-debug=1', "-keypool=100", ed2],
```

Comment:
This was a leftover from my local dev view. I have restored the previous debugging flags now.

I also agree we should investigate the impact of debugging logs in future, and in case reconsider our current usage.